From a33d2c099e5ebc8135cc03cf7a1153257d87a057 Mon Sep 17 00:00:00 2001
From: Mario Limonciello <mario_limonciello@dell.com>
Date: Wed, 15 Jul 2015 16:17:52 -0500
Subject: [PATCH 1/2] Allow fwupdmgr to issue offline 'install' commands.

Previously the only way to flash a UEFI capsule file was to issue a request
for update-offline along with the GUID and the filename.

This allows fwupdmgr to be launched with just the filename and it will
check if the GUID matches.  If it does, the update will be issued in offline
mode on the next reboot.
---
 src/fu-util.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/fu-util.c b/src/fu-util.c
index c7cae47..293f49e 100644
--- a/src/fu-util.c
+++ b/src/fu-util.c
@@ -493,7 +493,8 @@ fu_util_install (FuUtilPrivate *priv, gchar **values, GError **error)
 		return FALSE;
 	}
 	return fu_util_update (priv, FWUPD_DEVICE_ID_ANY,
-			       values[0], priv->flags, error);
+			       values[0], priv->flags | FU_PROVIDER_UPDATE_FLAG_OFFLINE,
+                               error);
 }
 
 /**
-- 
1.9.1

